woocommerce/woocommerce
# Revert #46362

### Changes proposed in this Pull Request:

This PR reverts #46362.

In #46362, we updated the styles for the Block-based checkout input fields, including adopting `currentColor` to inherit body text color. However, this lead to issues when using dark mode themes if the dark mode setting is not enabled in the Checkout block. Specifically, input fields may have a white background with white text, making the input values invisible.

### How to test the changes in this Pull Request:

1. Verify that all CI checks are passing.
2. Execute a smoke test of the Checkout block.
3. Verify that the text of the input fields remains visible under the following scenarios:
    - Using a dark mode theme, e.g. the _Maelstrom_ style of _Twenty Twenty-Four_, but do not enable the dark mode settings in the Checkout block.
    - Using a dark mode theme, e.g. the _Maelstrom_ style of _Twenty Twenty-Four_, and enable the dark mode settings in the Checkout block.
    - Using a light mode theme, e.g. the _Default_ style of _Twenty Twenty-Four_, and do not enable the dark mode settings in the Checkout block.

### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

This PR reverts #46262, as that PR would render input values invisible under certain conditions.

</details>
